docs/frameworks/laravel.md

```diff
@@ -49,20 +53,11 @@ functions:
             - ${bref:layer.console} # The "console" layer
 ```
 
-Now we still have a few modifications to do on the application to make it compatible with AWS Lambda.
-
-Since [the filesystem is readonly](/docs/environment/storage.md) except for `/tmp` we need to customize where the cache files are stored. Add this line in `bootstrap/app.php` after `$app = new Illuminate\Foundation\Application`:
-
-```php
-/*
- * Allow overriding the storage path in production using an environment variable.
- */
-$app->useStoragePath($_ENV['APP_STORAGE'] ?? $app->storagePath());
-```
```

Comment:
Thanks, the PR looks almost ready to merge.

Now I am wondering about this part that ends up being removed: is this intended?

In the PR we still define `APP_STORAGE=/tmp`, but the variable is not used anymore because this modification has been removed: we should probably keep it?

Reply:
no you are right. Lets simplify as much as possible. Its removed now.

Reply:
Sorry for the delay. I was actually expecting the opposite: are we sure that customizing the storage path is not necessary anymore?

Reply:
in laravel 6 its $_ENV['APP_BASE_PATH'] ?? dirname(__DIR__). so that can be used.